scripts/cloudant/manage_cloudant.py

```diff
@@ -8,6 +8,12 @@
 Auth = namedtuple('Auth', 'username password')
 
 
+class AskRun(namedtuple('AskRun', 'ask run')):
+    def ask_and_run(self):
+        ask_user(self.ask())
```

Comment:
Yeah I tested and this works. Since ask and run are set as functions at instantiation time (and not in the class definition), `self.ask` and `self.run` just return the function they were set to (as if it were any other variable) and not a bound method.